addons/web_unsplash/static/src/components/media_dialog/image_selector.xml

```diff
@@ -24,16 +24,14 @@
         <a href="https://www.odoo.com/documentation/16.0/applications/websites/website/optimize/unsplash.html#generate-an-unsplash-application-id"
            class="mx-2" target="_blank">Application ID</a>
         here:
-        <div class="input-group d-flex justify-content-end align-items-center w-auto mx-2">
-            <input type="text"
-                class="o_input form-control w-auto"
-                placeholder="Paste your application ID here"
-                t-model="state.appId"
-                t-on-input="() => this.state.hasAppIdError = false"
-                t-att-class="{ 'is-invalid': state.hasAppIdError }"/>
-            <button type="button" class="btn btn-primary btn-block w-auto p-auto save_unsplash" t-on-click="() => this.submitCredentials()">Apply</button>
-        </div>
+        <input type="text"
+            class="o_input form-control w-auto ms-2"
```

Comment:
I think we don't need the `form-control` class (on the previous input neither).
Also, I'd suggest to add `o_required_modifier` to both since are technically required to proceed.